libs/shared/src/lib/components/powerbi/ReportMeta.tsx

```diff
@@ -0,0 +1,65 @@
+import { Popover, Typography } from '@equinor/eds-core-react';
+import { ReportMetaDataProps } from '@equinor/workspace-fusion/power-bi';
+import { useQuery } from '@tanstack/react-query';
+import styled from 'styled-components';
+import { usePBIHelpers } from '../../hooks';
+
+export const ReportMeta = ({
+  anchor,
+  reportUri,
+  reportName,
+}: ReportMetaDataProps & { reportName: string }) => {
+  const { getReportInfo } = usePBIHelpers();
+  const { data } = useQuery(
+    ['reportInfo', reportUri, reportName],
+    ({ signal }) => getReportInfo(reportUri, signal),
+    {
+      suspense: true,
+      useErrorBoundary: true,
+    }
+  );
+
+  if (!data) {
+    throw new Error();
+  }
+  return (
+    <Popover
+      open
+      anchorEl={anchor}
+      title={`Report information ${reportName}`}
+      placement="bottom"

```

Comment:
Do we want to set a width here or just let it be based on the content of the popover?

Reply:
I tried letting it be dynamic but I found it to be weird in size

Reply:
Maybe minWidth would be viable